### DISCUSSION TITLE
New polymorphism features: "Diff warning" and "copy to live"

### DISCUSSION THREAD

**Discussion by rockgobo** [2sic/2sxc#3086] [Ideas]
Hallo SIC,
we started to work with polymorphism and the first thing we stumbled over was the fact that live and staging can indeed be different ... sounds obvious at first, but it would be helpfull if the UI could give a little hint if both "worlds" differ. It would also be handy, if there would be a little button to copy the code from staging to live. 

We are in a context where we a not able to connect to the live server with Git or visual Studio Code, so we depend on the tools build in  the UI of 2SXC. 

All the best 
  Christian

**Comment by iJungleboy** [comment]
Hi @rockgobo 

So basically these are two wishes:

1. Indication that you're working on a staging edition in the output
2. tools to optimize publishing changes

The indication sounds fairly simple, and like a good idea. Not yet sure how to do it, but it should be easy. 

The tools would require quite a bit more work; ideally with diff-tools and wizard or something. that can easily be 3-5 days of work so I'm sure we won't have the resources to do it unless someone sponsors the work.
